src/config.cc

```diff
@@ -164,6 +164,15 @@ void Config::initFieldValidator() {
         if (v.empty() && !tokens.empty()) {
           return Status(Status::NotOK, "requirepass empty not allowed while the namespace exists");
         }
+        if (tokens.find(v) != tokens.end()) {
```

Comment:
This validation should move to the callback since the validator would be executed before parsing the entire config file. So tokens array would be empty when requirepass was parsed before those token items.

Reply:
Indeed, but I think this behavior is a validate behavior, and it's not elegant to put it in a callback.

Can we modify the `Config::parseConfigFromString()`, remove the inside of the validate behavior, And then validate in the `Config::Load`(just like the callback behavior).

Reply:
Yes, it would be better, go ahead.

Reply:
New PR or in this PR?

Reply:
you can do it in this PR since it is internal refactor instead of influencing user's usage.